root/static/scripts/edit/externalLinks.js

```diff
@@ -480,6 +477,10 @@ export class ExternalLink extends React.Component<LinkProps> {
                   {l('video')}
                 </label>
               </div>}
+            {props.errorMessage &&
+              <div className="error field-error" data-visible="1">
+                {props.errorMessage}
+              </div>}
```

Comment:
This is generally good, but: 
![Screenshot from 2021-05-25 18-48-26](https://user-images.githubusercontent.com/1069224/119528432-d6fce800-bd89-11eb-8f9c-a237d036bc04.png)
This use case shows a situation where it feels off - it seems like the required field would be the (already filled in) type, but what it means is the actual URL. Can we add the error to the appropriate input (type if it fails a validation, text if it's an error unrelated to the type) instead, and have both options possible? :)

Reply:
Makes sense, I'll dig into the code later to see if we can realize it.

Reply:
This would be nice to have but can probably be added later on.
It requires a lot of rework to make errors be more specific about which input(s) is (are) in cause.
This is probably worth a separate ticket.